### PR TITLE
fix: check payment status field

### PR DESCRIPTION
**Pull request by motorina0**
Depends on: https://github.com/lnbits/lnbits/pull/2537
Expected to fail until the above PR is merged.

### CODE REVIEW

File: integration/005_lnurlw-race.jmx [hunk 1]
```diff
@@ -2154,7 +2154,7 @@ if (resp.length !== parallelWithdrawUsers) {
   var withdrawOkCount = 0;
 
   for (var i = 0; i &lt; resp.length; i++) {
-    if (resp[i].pending === false) {
+    if (resp[i].status === &quot;success&quot;) {
```
Comment (prusnak) [inline integration/005_lnurlw-race.jmx:2157]: lol :)

**Review by prusnak — CHANGES_REQUESTED** [review]

File: integration/005_lnurlw-race.jmx (hunk 1 above)
Comment (dni) [reply integration/005_lnurlw-race.jmx:2157]: :dagger:

Comment (motorina0) [reply integration/005_lnurlw-race.jmx:2157]: extracted JS assertions to separate file.
see https://github.com/lnbits/lnbits-extensions/pull/365

**Comment by motorina0** [comment]
moved here: https://github.com/lnbits/lnbits-extensions/pull/365

**Pull request closed without merging**